libfreerdp/core/proxy.c

```diff
@@ -490,9 +490,10 @@ static int recv_socks_reply(BIO* bufferedBio, BYTE* buf, int len, char* reason,
 		status = BIO_read(bufferedBio, buf, len);
 
 		if (status > 0)
-			break;
```

Comment:
Why brace single line statements, that is not our style ;)

Reply:
Ops! Sorry, i fixed it.